modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ues/ui/integration/test/dashboard/BannerTest.java

```diff
@@ -0,0 +1,384 @@
+/*
+ * Copyright 2005-2015 WSO2, Inc. (http://wso2.com)
+ *
```

Comment:
Having date range is no valid as per the new license policy standards.. Please only the year when source code is originated..